Bottom: 9c6276fb1f4a9f78faca3cb229042bc3d99c77b9
Top:    fcc922b3867725a607261c11f3e65da207856e70
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-03 16:27:46 +0800

Refresh of debug.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 45ca99b..252a49e 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1343,7 +1343,7 @@ static void *statemachine(void *arg)
 		case SIGUSR2:
 			master_mutex_lock();
 			if (list_empty(&master_list->completed)) {
-				if (list_empty(&master_list->mounts)) {
+				if (!list_empty(&master_list->mounts)) {
 					debug(LOGOPT_ANY, "list empty completed and mounts");
 					master_mutex_unlock();
 					return NULL;
